### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by ghost** [Galts-Gulch/avarice#27]
Would it be possible to add in arbitrage for that extremely RARE occasion when there is profit to be made using LTC, BTC, and Yuan?

**Comment by RealJohnGalt** [comment]
It would certainly be possible, but there's currently no support for switching the tradepair on the fly. This would also require a new strategy, since it couldn't be handled by a new indicator alone.

**Comment by ghost** [comment]
Correct, it would require having it look at all 3 trading pairs and checking to see if it would be profitable to switch btc to ltc to yuan or vice versa. I used to do that quite a bit on btc-e, but with a trading fee of 0.2% it made it difficult to do it by hand. 

While searching the name of the bot that I can't remember that used to do btc-e arbitrage trading I came across a python bot that already does and is open source. I don't want to post the link here, but if you Private Message me, or email me I'll send you the link.

**Comment by RealJohnGalt** [comment]
Thank you for your offer, I believe I actually have a copy of the bot you're talking about. While working on the FIX API implementation I decided to also redo the websocket implementation. The new code will make arbitrage easier to support and uses asyncore backend.
